## 03_check_snmp_load by Jan Wagner <waja@cyconet.org>
##
## DP: Update script to version 1.12

diff -Nur nagios-snmp-plugins-1.1.1.orig/check_snmp_load.pl nagios-snmp-plugins-1.1.1/check_snmp_load.pl
--- nagios-snmp-plugins-1.1.1.orig/check_snmp_load.pl	2007-04-16 19:41:20.000000000 +0200
+++ nagios-snmp-plugins-1.1.1/check_snmp_load.pl	2007-10-12 22:23:23.000000000 +0200
@@ -1,11 +1,10 @@
 #!/usr/bin/perl -w 
 ############################## check_snmp_load #################
-# Version : 1.3.2
-# Date : Jan 16 2007
+my $Version='1.12';
+# Date : Oct 12 2007
 # Author  : Patrick Proy ( patrick at proy.org)
-# Help : http://www.manubulon.com/nagios/
+# Help : http://nagios.manubulon.com/
 # Licence : GPL - http://www.fsf.org/licenses/gpl.txt
-# Changelog : HP-UX load added.
 # Contributors : F. Lacroix and many others !!!
 #################################################################
 #
@@ -18,10 +17,8 @@
 
 # Nagios specific
 
-use lib "/usr/local/nagios/libexec";
-use utils qw(%ERRORS $TIMEOUT);
-#my $TIMEOUT = 15;
-#my %ERRORS=('OK'=>0,'WARNING'=>1,'CRITICAL'=>2,'UNKNOWN'=>3,'DEPENDENT'=>4);
+my $TIMEOUT = 15;
+my %ERRORS=('OK'=>0,'WARNING'=>1,'CRITICAL'=>2,'UNKNOWN'=>3,'DEPENDENT'=>4);
 
 # SNMP Datas
 
@@ -94,7 +91,6 @@
 
 # Globals
 
-my $Version='1.3.2';
 
 my $o_host = 	undef; 		# hostname
 my $o_community = undef; 	# community
